model/base/src/main/java/io/sundr/model/Property.java

```diff
@@ -32,18 +32,23 @@ public class Property extends ModifierSupport implements Renderable, Commentable
   private final TypeRef typeRef;
   private final String name;
   private final List<String> comments;
+  private final boolean enumConstant;
+  private final boolean synthetic;
 
-  public Property(List<AnnotationRef> annotations, TypeRef typeRef, String name, List<String> comments, Modifiers modifiers,
```

Comment:
I think that you should keep the old constructor around for compatibility reasons.
Make sure the old constructor is after the new so that builders get generated with the new one (I think the first non empty is picked or something like that).

Reply:
done, thanks for the suggestions!